toolchain/lower/file_context.h

```diff
@@ -165,6 +184,69 @@ class FileContext {
 
   auto BuildVtable(const SemIR::Class& class_info) -> llvm::GlobalVariable*;
 
+  // Track which specifics were lowered for each generic. These are added one
```

Comment:
```suggestion
  // Records a specific that was lowered for a generic. These are added one
```

I think `lowered_specifics_`, the member variable, could be described as tracking all specifics. This function just records a single specific.